Phprojekt 350

exi opened the pull request:
fix submodule bugs

exi committed:
checkstyle

exi committed:
fix rendering of submodules

the submodule rendering did not use the delayed rendering process like the
normal form.
we now just pass in the outer form and use the outer forms addTab method to
add the submodule.
This still has some issues like dynamic sizing but it will be much more work
to fix these.

exi committed:
fix form width

the form width was 80% which was wrong

exi committed:
fix submodule buttons

the type="submit" triggered a form submit on every click, deleting it fixes the
issue

voidus commented:
Code looks good but this doesn't seem to fix the problem. See the bug for a screenshot.

exi committed:
fix minutes submodule rendering

we inserted malformated html in the page, namely div elements into a <table>

exi committed:
checkstyle

exi committed:
fix form segment rendering in minutes

i found another error similar to the one previously fixed in the mail section.
we now split it up into several tr's an add them as widgets to the section.

exi commented:
updated, should work now
